### PR TITLE
adding range support to number of likes per interaction

### PR DESCRIPTION
**Pull request by philip-ulrich**
Not sure if you want to merge this or not, but I think it's useful and doesn't add any more parameters or anything. Just allows you to specify a range of likes and randint will choose one at "random".

### CODE REVIEW

**Comment by philip-ulrich** [comment]
> Support intervals for likes and interactions count like --likes-count 2-3
I just noticed that this was in your list of desired features. Would it be helpful if I added this same functionality to other inputs as well? Like number of interactions?

**Comment by alexal1** [comment]
@philip-ulrich amazing PR, sorry for not reviewing it earlier! Yes sure, that'd be good to have the same feature for interactions!

**Comment by philip-ulrich** [comment]
Sure. I'll try to get that change made tonight.

**Comment by philip-ulrich** [comment]
@alexal1: All the parameters support range now! (except unfollow actions - it'd be easy to add, but it doesn't seem like it'd be incredibly useful and I don't use them enough to test it)

**Comment by philip-ulrich** [comment]
I'd suggest testing it yourself, but the logic is fairly simple and it's working for me. 😄

**Comment by philip-ulrich** [comment]
Bump? 😄 I'd like to not have to keep updating my fork to get the latest fixes on my copy - especially if these are features you want.

File: insomniac.py [hunk 1]
```diff
@@ -78,10 +78,15 @@ def main():
             print("Action: remove " + str(args.remove_mass_followers) + " mass followers")
             mode = Mode.REMOVE_MASS_FOLLOWERS
 
+    args.interactions_count = get_count(args.interactions_count, 'Interactions count', 70)
```
Comment (alexal1) [inline insomniac.py:81]: Why do we need these lines?
It's better not to touch args, as they are our script's input.

Comment (alexal1) [reply insomniac.py:81]: Btw calculating `get_count()` on this stage is the same as to just pass some another value to the script, isn't it?

**Comment by alexal1** [comment]
Merged your PR with some minor improvements here b9497bf92e2f30850b4f80a5897de31f4f21a562
Thanks for your contribution!

**Pull request closed without merging**